### ISSUE TITLE
Error for '/' character in filename

### ISSUE THREAD

**Issue by LoveCany** [maple3142/GDIndex#172]
# Bug Description / 問題描述

When the filename includes '/' character, the file/folder can never be opened correctly.

![image.png](https://i.loli.net/2020/11/18/hxbU5kKuE6zw7cI.png)

The full name for this folder should be "[BDMV][201118][BRMM-10310][Roselia×RAISE A SUILEN「Rausch und/and Craziness」]", but GDIndex seems to separate the filename into two. This still doesn't work even if I change the '/' in URL into %2F.

# Environment / 環境

* Browser version / 瀏覽器版本:
Chrome 86
* OS version / 作業系統版本:
Windows 10 Version 2004

**Comment by maple3142** [comment]
It is better to avoid "/" in filename

**Comment by ztgasdf** [comment]
True, but in terms of handling files that *do* have slashes in their name, there should be something to allow them to be accessed.

**Comment by maple3142** [comment]
You can just rename those files. This restriction is reasonable as Windows and Linux both doesn't allow having forward slash in filename.
